presto-main/src/main/java/io/prestosql/server/testing/TestingPrestoServer.java

```diff
@@ -229,6 +233,8 @@ public TestingPrestoServer(
                     binder.bind(TestingEventListenerManager.class).in(Scopes.SINGLETON);
                     binder.bind(AccessControlManager.class).to(TestingAccessControlManager.class).in(Scopes.SINGLETON);
                     binder.bind(EventListenerManager.class).to(TestingEventListenerManager.class).in(Scopes.SINGLETON);
+                    binder.bind(PrestoGroupProviderManager.class).in(Scopes.SINGLETON);
```

Comment:
This can be removed

Reply:
It's needs to be injected into PluginManager to allow registering different group providers.